.github/workflows/publish.yml

```diff
@@ -70,7 +119,6 @@ jobs:
           CHROME_REFRESH_TOKEN: ${{ secrets.CHROME_REFRESH_TOKEN }}
       - name: Expose Chrome extension artifact
         if: '!inputs.prerelease'
-        continue-on-error: true
```

Comment:
This is because of the following step in the dev process:
- Kilian wanted to test quickly his deployment
- So he parallelized the deployment task
- But it is not possible to parallelize the deployment
- So he split the job to several ones he parallelized
- Now that there is only one task in the job, no need to "continue on error" be cause there is nothing to continue too
- Kilian removed this line
(- Kilian likes to talk about Kilian like Sinok in the Goonies)

Reply:
It is actually meaningless to have  `continue-on-error` on the last instruction task of a job.
The change has been also extracted in #319